cmd/cmd/gateway.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/spf13/cobra"
 
+	cosmoslp "github.com/tendermint/cosmos-rosetta-gateway/generated/cosmos-launchpad"
```

Comment:
is it possible to insert the generated inside rosetta/gateway/cosmos/launchpad?

Reply:
I would do some changes with packages, I would move the generated into the package for launchpad. 

Later the mocks one, that is related too with the specific specification of launchpad generated must live too inside the rosetta/gateway/cosmos/launchpad somewhere.

And now that I see another one I would try to remove in one moment or other is the util one, in go is somehow like an antipattern. Let me give you some reference and then you decide the best way.

https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
Apart from that the generated API client is a nice improvement!! Good job!!!

Reply:
Hey thanks Jonathan! I made some changes on folder structure, please check f95a741.

Dave's articles really good :) I agree on util is a bad name, I usually use `x`  (extend) prefix to extend standard packages  functionalities. But `x` is a reserved name in Cosmos so we cannot use. Maybe we can give a unique name to those packages, which might be the best. But for now I'm leaving this one as it is because after having a generated client for Tendermint, we will remove `type.go` and the `util` pkg.